Aquality.Selenium.Core/src/Aquality.Selenium.Core/Forms/Form.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Elements;
+using Aquality.Selenium.Core.Elements.Interfaces;
+using Aquality.Selenium.Core.Localization;
+using Aquality.Selenium.Core.Visualization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Aquality.Selenium.Core.Forms
+{
+    /// <summary>
+    /// Describes form that could be used for visualization purposes (see <see cref="Dump"/>).
+    /// </summary>
+    /// <typeparam name="T">Base type(class or interface) of elements of this form.</typeparam>
+    public abstract class Form<T> : IForm where T : IElement
+    {
+        /// <summary>
+        /// Name of the current form.
+        /// </summary>
+        public abstract string Name { get; }
+
+        /// <summary>
+        /// Visualization configuration used by <see cref="Dump"/>.
+        /// Could be get from AqualityServices.
```

Comment:
A slight typo - should be `could be gotten`. However, I would prefer `Could be obtained`.

Reply:
Seemingly, needs to be updated further as well.

Reply:
updated to "Could be obtained"